tests/adiabatic_heating_with_melt/screen-output

```diff
@@ -24,7 +24,7 @@ Number of degrees of freedom: 9,825 (2,898+1,365+2,898+405+1,449+405+405)
 
    Postprocessing:
      Temperature min/avg/max:       1723 K, 1755 K, 1788 K
-     Heating rate (average/total):  0 W/kg, 0 W
+     Heating rate (average/total):  -3.191e-11 W/kg, -177.4 W
```

Comment:
This is a significant change, and it is not related to the problem in the visualization postprocessor. Is that expected?

Reply:
This is now correct, and was wrong before.